src/Sylius/Bundle/ApiBundle/CommandHandler/Payment/UpdatePaymentRequestHandler.php

```diff
@@ -30,10 +30,11 @@ public function __construct(
     public function __invoke(UpdatePaymentRequest $updatePaymentRequest): PaymentRequestInterface
     {
         $hash = $updatePaymentRequest->getHash();
-        Assert::notNull($hash, 'Payment request hash cannot be null.');
 
         $paymentRequest = $this->paymentRequestRepository->findOneByHash($hash);
```

Comment:
Line 
```php
$hash = $updatePaymentRequest->getHash();
```
is no longer needed, if `getHash()` is longer nullable.

```suggestion
        $paymentRequest = $this->paymentRequestRepository->findOneByHash($updatePaymentRequest->getHash());
```